community/templates/meetups/attend.html

```diff
@@ -1,4 +1,4 @@
-{% extends "base.html" %}
+{% extends "base_user.html" %}
```

Comment:
That is probably right. If it isn't, no problem. I can easily change it afterwards